src/app/components/auth/signup/signup.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="signup-container">
   <app-input [isRequired]="true" [label]="'Username'" [type]="'text'" [theme]="'dark'" [icon]="'assets/images/username_dark.png'" #formsignup></app-input>
   <app-input [isRequired]="true" [label]="'Email'" [type]="'email'" [theme]="'dark'" [icon]="'assets/images/envelope_dark.png'" #formsignup></app-input>
-  <app-input [isRequired]="true" [label]="'Password'" [type]="'password'" [theme]="'dark'" [icon]="'assets/images/password_dark.png'" #formsignup></app-input>
+  <app-input [isRequired]="true" [label]="'Password1'" [type]="'password'" [theme]="'dark'" [icon]="'assets/images/password_dark.png'" #formsignup></app-input>
```

Comment:
Please elaborate a bit about these changes. Also let us know other scenarios like alphabet only, Capital letter, special character etc.

Reply:
I've checked other scenarios and it's breaking on entirely numeric password for me.

So, when the password is entirely numeric the server returns the error with the key "password1", now the problem is that there's a function called `formValueForLabel` whose job is to fetch the form field against the given key, since the submission has failed and we're in the error handling part, when given "password1" as the key the function fails to fetch the field since the label of the field is "password" and not "password1". That's why the error was not showing for entirely numeric password.